internal/unknown/path_error.go

```diff
@@ -0,0 +1,33 @@
+package unknown
+
+import (
+	"regexp"
+
+	"github.com/anchore/syft/internal/log"
+	"github.com/anchore/syft/syft/file"
+)
+
+var pathErrorRegex = regexp.MustCompile(`.*path="([^"]+)".*`)
```

Comment:
Can we look at the error type instead of the text like this?

Reply:
We syned on this offline, we're going to follow up with another PR that has more broad impact/fixes